social_core/backends/open_id_connect.py

```diff
@@ -29,6 +29,16 @@
 
     from requests.auth import AuthBase
 
+import hashlib
+
+try:
+    from cryptography.hazmat.backends import default_backend
+    from cryptography.hazmat.primitives import hashes
+
+    has_crypto = True
+except ModuleNotFoundError:
+    has_crypto = False
```

Comment:
It should be always there because of:

https://github.com/python-social-auth/social-core/blob/75c14bb9983e0fa1b7af3e9b8b25db48ced6d157/pyproject.toml#L47